OpenUpSA/wazimap-ng-ui
# negative range

## Description
bar charts and choropleth can display negative values now

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-571

## How is it tested automatically?
added `__tests__/elements/map/choropleth.test.js`

## How should a reviewer test it locally
* upload a dataset that contains negative values
* confirm that the choropleth, choropleth legend and the charts are displayed correctly

## Screenshots
![image](https://user-images.githubusercontent.com/53019884/204539102-ded5ee8a-67e6-4742-980e-e3b1f810465b.png)
![image](https://user-images.githubusercontent.com/53019884/204539267-e3237087-0c19-40ef-8dad-f1de04b00b3a.png)


## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
